src/ranged.cpp

```diff
@@ -1221,7 +1221,7 @@ std::vector<tripoint> game::pl_target_ui( target_mode mode, item *relevant, int
                            m.mode.c_str(), m.qty );
             }
 
-            auto cur = ammo ? ammo : m->ammo_data();
+            const itype *cur = ammo ? ammo : m->ammo_data();
```

Comment:
huh, I'm actually talking about stuff two lines below  (```auto str = string_format( m->ammo_remaining() ?```) but it's a good change anyway ;-)